### PR TITLE
Consent transparency

### PR DESCRIPTION
**Pull request by ulrikus**
# What?
Adds a `ConsentTransparencyView` meant to show **once** at first startup fra all users.

Todo:
- [x] Test on iPhone 4, since there is no scrolling. Might have to add it.

# Why?
It's the newest hot potato from a few steps up the GDPR ladder. A way of priming/informing the users of _BLINK_.

# Screenshots
### iPhone X vs iPhone 5
iPhoneX iPhone5

### iPad
iPad

### CODE REVIEW

File: Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    // MARK: - Superclass Overrides
+
+}
+
+// MARK: - Public
```
Comment (axldyb) [inline Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift]: Maybe remove unused extensions and MARKs?

File: Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Din data, dine valg"
+        return label
+    }()
+
+    private lazy var detailLabel: Label = {
+        let label = Label(style: .detail(.licorice))
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.text = "FINN er en del av Schibsted Norge. Når du bruker FINN er Schibsted Norge behandlingsansvarlig for påloggingsløsning og reklame, mens FINN er behandlingsansvarlig for det øvrige innholdet i tjenesten vår. Både FINN og Schibsted Norge behandler data om deg.\n\nFINN bruker dine data til å tilpasse tjenestene til deg, mens Schibsted Norge i tillegg bruker dem til å gi deg mer relevante annonser. Persondata brukes også for å sikre at tjenestene er trygge og sikre for deg."
```
Comment (axldyb) [inline Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift]: We add these texts from a model in other components. Now they are hardcoded. Any reason not to follow that pattern?

File: Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift (hunk 1 above)
Comment (ulrikus) [reply Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift]: I agree. Done! ✅

File: Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+            headerLabel.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: .mediumLargeSpacing),
+            headerLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .mediumLargeSpacing),
+            headerLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.mediumLargeSpacing),
+
+            detailLabel.topAnchor.constraint(equalTo: headerLabel.bottomAnchor, constant: .mediumSpacing),
+            detailLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .mediumLargeSpacing),
+            detailLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.mediumLargeSpacing),
+            detailLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -.smallSpacing),
+
+//            buttonStackView.topAnchor.constraint(equalTo: detailLabel.bottomAnchor, constant: .largeSpacing),
```
Comment (axldyb) [inline Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift]: Remove this commented line if we are not using it?

Comment (ulrikus) [reply Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView.swift]: Fixed! ☑️

**Pull request merged**